ogr/services/github/service.py

```diff
@@ -59,6 +60,7 @@ def __init__(
         github_app_private_key_path: str = None,
         tokman_instance_url: str = None,
         github_authentication: GithubAuthentication = None,
+        max_retries=0,
```

Comment:
What about having some constant for this and use here and in the `PagureService` as well?

Reply:
The question is, if we want to have the same retry policy for `GithubService` and `PagureService`.

It shouldn't be a problem to have a constant defined for both services, but in that case I think, it should  be an integer rather than  `urllib3.util.Retry`. Currently the `Retry` object for `GithubService` is constructed [here](https://github.com/packit/ogr/blob/39a6b4985490d719e2a0f18d69b0034e5a949fb5/ogr/services/github/service.py#L76), and quite defensive strategy was chosen for now. It retries only in specific cases (only HTTP status 401, 403, 500; only certain HTTP methods; only certain error types).

Reply:
OK, it makes sense to have different defaults.